### PR TITLE
[stable/mlflow] create mlflow chart initially

### PR DESCRIPTION
**Pull request by lena-kuhn**
## Checklist

- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)
- [x] I have read the [contribution instructions](https://github.com/deliveryhero/helm-charts#contributing), bumped chart version and regenerated the docs
- [x] Github actions are passing

### CODE REVIEW

File: stable/mlflow/templates/database.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: postgresql.org/v1
```
Comment (max-rocket-internet) [inline stable/mlflow/templates/database.yaml:1]: This needs to be optional, so let's add a condition around it, something like this:

```suggestion
{{ if .Values.mflow.postgres.PostgresDatabaseEnabled -}}
apiVersion: postgresql.org/v1
```

File: stable/mlflow/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+name: mlflow
+description: A Helm chart for MLflow
+type: application
+version: 1.0.0
+home: https://www.mlflow.org/
+appVersion: "latest"
```
Comment (max-rocket-internet) [inline stable/mlflow/Chart.yaml:7]: It's bad practice to use `latest` so how about use the latest proper tag, 1.9.1?

https://hub.docker.com/layers/larribas/mlflow/1.9.1/images/sha256-05ccb0b46bfb60ff0f88ed0bb0fa006a6e581e2d537f13ebebfeb98d3a13b90b?context=explore

File: stable/mlflow/values.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
+# Default values for mlflow.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: larribas/mlflow
+  pullPolicy: IfNotPresent
+  tag: latest
```
Comment (max-rocket-internet) [inline stable/mlflow/values.yaml:8]:
```suggestion
  tag: 1.9.1
```

File: stable/mlflow/templates/database.yaml (hunk 1 above)
Comment (lena-kuhn) [reply stable/mlflow/templates/database.yaml:2]: Good point, thanks :)!

File: stable/mlflow/Chart.yaml (hunk 2 above)
Comment (lena-kuhn) [reply stable/mlflow/Chart.yaml:7]:
```suggestion
appVersion: 1.9.1
```

File: stable/mlflow/templates/service.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "mlflow.fullname" . }}
+  labels:
+    {{- include "mlflow.labels" . | nindent 4 }}
+  annotations:
```
Comment (nyambati) [inline stable/mlflow/templates/service.yaml:7]: Remove this annotation, its no longer relevant

**Review by nyambati — CHANGES_REQUESTED** [review]

File: stable/mlflow/templates/service.yaml (hunk 4 above)
Comment (lena-kuhn) [reply stable/mlflow/templates/service.yaml:7]: True, good catch! removed

**Review by nyambati — APPROVED** [review]

**Pull request merged**